.gitignore

```diff
@@ -122,3 +122,6 @@ built-docs
 
 # MacOS X
 .DS_Store
+
+# sqlite
+storage.db
```

Comment:
this should not be created in the repo directory right? curious why adding this

Reply:
if I install BentoML by `pip install -e .` then this file will be generated when I run a service.save() command.